osgar/logger.py

```diff
@@ -190,6 +192,74 @@ def write(self, stream_id, data):
             pass
 
 
+import mmap
+
+def _create_index(data, pos, start_time=datetime.timedelta()):
+    index = []
+    end = len(data)
+    times, prev = divmod(start_time, datetime.timedelta(microseconds=TIMESTAMP_OVERFLOW_STEP))
+    us_offset = times * TIMESTAMP_OVERFLOW_STEP
+    prev_micros = prev.microseconds + prev.seconds * 10**6 + prev.days * 24 * 3600 * 10**6
+    while pos+8 < end:
```

Comment:
it is funny, that here I would use extra spaces "pos + 8", while a bit lower I would remove space from "datetime.timedelta(microseconds=micros+us_offset)"

Reply:
fixed (here and also the timedelta)